dist/tools/usb-cdc-ecm/start_network.sh

```diff
@@ -57,25 +56,42 @@ cleanup() {
 }
 
 start_uhcpd() {
+    ip route add "${PREFIX}" via fe80::2 dev "${INTERFACE}"
     ${UHCPD} "${INTERFACE}" "${PREFIX}" > /dev/null &
     UHCPD_PID=$!
 }
 
 start_dhcpd() {
+    ip route add "${PREFIX}" via fe80::2 dev "${INTERFACE}"
     DHCPD_PIDFILE=$(mktemp)
     ${DHCPD} -d -p "${DHCPD_PIDFILE}" "${INTERFACE}" "${PREFIX}" 2> /dev/null
 }
 
+start_radvd() {
+    ADDR=$(echo "${PREFIX}" | sed -e 's/::\//::1\//')
+    ip a a "${ADDR}" dev "${TAP}"
```

Comment:
Ups, it was called `INTERFACE` here, sorry.
```suggestion
    ip a a "${ADDR}" dev "${INTERFACE}"
```